docs/infra/identity-provider.md

```diff
@@ -0,0 +1,35 @@
+# Identity Provider
```

Comment:
This could be a followup thing, but it might be worth noting something about if you have multiple applications that want to use the same user pool, recommended (high-level) setup for that? Not sure what it'd be, create a "login" application/service that the other apps can use as an SSO?

Reply:
Good call out. I can think of a couple of ideas, but we haven't actually implemented any of them so duno how feasible they are:

1. create a separate app just for login, kind of like a login.gov. pros: it can later be pulled out into it's own repo to act as the agency's SSO if they don't have one
2. enable the id provider in one app, and add an IAM policy to allow the other app to access the id provider, and add environment variables to the other app to be able to access it

Reply:
Created ticket for now https://github.com/navapbc/template-infra/issues/912